migration/src/m0000870_source_doc_indexes.rs

```diff
@@ -0,0 +1,87 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+#[allow(deprecated)]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .create_index(
```

Comment:
thx for this - I believe we had these before and after the refactoring into source_document .. not sure when/why this happened - glad they are back.